src/bin/pg_autoctl/ipaddr.c

```diff
@@ -133,6 +133,19 @@ fetchLocalCIDR(const char *localIpAddress, char *localCIDR, int size)
 		char netmask[INET6_ADDRSTRLEN] = { 0 };
 		char address[INET6_ADDRSTRLEN] = { 0 };
 
+		/*
+		 * Some interfaces might have an empty ifa_addr, such as when using the
```

Comment:
is there a way to test this ?

Reply:
It sounds complex to build a test case for this and easy enough a code path to accept it without tests? better continue the look than crashing (segfault) on a NULL value anyway...